keyring_windows.go

```diff
@@ -0,0 +1,35 @@
+// +build windows
+
+package keyring
+
+import (
+	"github.com/danieljoos/wincred"
+)
+
+type winProvider struct {
+}
+
+func (p *winProvider) Get(Service, Username string) (string, error) {
+	cred1, err := wincred.GetGenericCredential(Service)
+	if err == nil && cred1.UserName == Username {
+		return string(cred1.CredentialBlob), nil
+
+	}
+	cred2, err := wincred.GetDomainPassword(Service)
```

Comment:
Hello mattn,

Given that one can add  a domain and generic credentials with the same target name (in this case, Service), it  would give incorrect results. If a user doesn't have generic credential with a given name but a domain credential they might receive it and I don't think it's a good idea. Also, considering that Set method sets generic credentials, it would be cleaner to do that in Get as well, unless if we plan other methods for domain credentials.

Let me know your ideas.

Thank you
Challa Rao Ande

Reply:
Thanks your comment. As you mentioned, it should have an issue. I'll look into it in later.

Reply:
I’d love to get this moving forwards. How can I help?